doc/classes/Callable.xml

```diff
@@ -13,21 +13,21 @@
 		    prints(arg1, arg2, arg3)
 
 		func test():
-		    callable.call("hello", "world")  # Prints "hello world".
+		    callable.call("hello", "world")  # Prints "hello world ".
 		    callable.call(Vector2.UP, 42, callable)  # Prints "(0, -1) 42 Node(Node.gd)::print_args".
 		    callable.call("invalid")  # Invalid call, should have at least 2 arguments.
 		[/gdscript]
 		[csharp]
-		Callable callable = new Callable(this, nameof("printArgs"));
+		Callable callable = new Callable(this, nameof(PrintArgs));
 		public void PrintArgs(object arg1, object arg2, object arg3 = "")
```

Comment:
```suggestion
		public void PrintArgs(object arg1, object arg2, object arg3 = null)
```
My bad.

Reply:
Well, `null` will be printed as `Null`, won't it? Anyway, C# example will probably need to be changed after `Callable` is implemented properly so I guess it shouldn't matter.

Reply:
You can't assign a string as default argument to an object. It won't compile.

Reply:
Yeah, I know (just didn't spot it originally). I meant in this specific case you'd rather not want that null to be printed at all. So for example:
```
public void PrintArgs(params object[] args)
```
would make more sense. But again: it doesn't really matter for now.

Are you fine with the changes made?